src/modules/organisation/persistence/organisation.entity.ts

```diff
@@ -42,4 +42,7 @@ export class OrganisationEntity extends TimestampedEntity {
     @AutoMap(() => String)
     @Enum({ items: () => Traegerschaft, nullable: true, nativeEnumName: 'traegerschaft_enum' })
     public traegerschaft?: Traegerschaft;
+
+    @Property({ nullable: true })
+    public emaildomain?: string;
```

Comment:
Sollte das nicht camelCase sein oder ist 'emaildomain' wirklich als ein Wort gedacht?
Falls camelCase -> überall einheitlich angepassen.

Reply:
Es ist nun camelCase, im Repo gibt es jetzt dafür eine Methode "mapEntityWithUnderScoreAttributesToAggregate", da das Mapping von z.B. email_domain zu emailDomain nicht wie gewohnt "out-of-the-box" funktioniert, wenn  man custom queries nutzt. Ich habe deshalb auch administriertVon und zugehoerigZu in der Methode behandelt. 
Ob andere Methoden nun angepasst werden müssen habe ich nicht bewertet, diese scheinen ja zu funktionieren wie gewünscht.